homeassistant/components/osc/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "osc",
+  "name": "OSC (Open Sound Control)",
+  "codeowners": ["@gmint"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/osc",
+  "homekit": {},
+  "integration_type": "hub",
+  "iot_class": "local_push",
```

Comment:
this appears to be `local_poll` not push, because the `update` function is periodically being called